storage/ndb/src/kernel/blocks/dblqh/DblqhMain.cpp

```diff
@@ -18907,15 +18980,30 @@ void Dblqh::storedProcConfScanLab(Signal* signal,
     signal->theData[1] = RNIL;
     signal->theData[2] = NextScanReq::ZSCAN_NEXT;
     signal->theData[0] = sig0;
+    // Moz statemach
+    ndbrequire(!scanPtr->m_aggregation ||
+               scanPtr->scanState == ScanRecord::WAIT_ACC_SCAN);
     scanPtr->scanState = ScanRecord::WAIT_NEXT_SCAN;
     scanPtr->scan_lastSeen = __LINE__;
+    // MOZ DEBUG PRINT
+    bool print = false;
+#ifdef MOZ_AGG_DEBUG
+    {
+    FragrecordPtr regFragptr;
+    regFragptr.i = tcConnectptr.p->fragmentptr;
+    ndbrequire(c_fragment_pool.getPtr(regFragptr));
+    if (regFragptr.p->fragId == 0 && scanPtr->m_aggregation) {
+      print = true;
+    }
+    }
+#endif // MOZ_AGG_DEBUG
     if (likely(in_send_next_scan == 0))
     {
       send_next_NEXT_SCANREQ(signal,
                              block,
                              f,
                              scanPtr,
-                             tcConnectptr.p->clientConnectrec);
+                             tcConnectptr.p->clientConnectrec, print);
```

Comment:
print on its own line to make it standardised

Reply:
ok